src/emulator/storage/rules/manager.ts

```diff
@@ -65,22 +59,16 @@ class DefaultStorageRulesManager implements StorageRulesManager {
     this._rules = _rules;
   }
 
-  start(): Promise<StorageRulesIssues> {
-    return this.updateSourceFile(this._rules);
+  async start(): Promise<StorageRulesIssues> {
+    const issues = await this.loadRuleset();
+    this.updateWatcher(this._rules.name);
+    return issues;
   }
 
   getRuleset(): StorageRulesetInstance | undefined {
     return this._ruleset;
   }
 
-  async updateSourceFile(rules: SourceFile): Promise<StorageRulesIssues> {
-    const prevRulesFile = this._rules.name;
-    this._rules = rules;
-    const issues = await this.loadRuleset();
-    this.updateWatcher(rules.name, prevRulesFile);
```

Comment:
I don't see where the `prevRulesFile` arg is being used anymore

Reply:
Good catch, we can get rid of it